homeassistant/components/slack/__init__.py

```diff
@@ -59,4 +62,24 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         )
     )
 
+    hass.config_entries.async_setup_platforms(
+        entry, [platform for platform in PLATFORMS if platform != Platform.NOTIFY]
```

Comment:
Why aren't we including `Platform.NOTIFY` in platform setup?

Reply:
I don't think platform is set up that way